USB Vendor and Product ID request for open HW project

lukaso25 opened the pull request:
Dear Open Moko team,

I would like to ask you for the USB Vendor and Product ID for my open HW project.

My USB device called "uDAC stereo audio DA converter UAC1 24/96" is a high quality stereo asynchronous USB audio digital to analog converter with 3.5 mm TRS jack output based on STM32F0 and CS43198.

This open hardware design is licensed under CERN Open Hardware License Version 2 - Weakly Reciprocal.

Link to the repository is https://bitbucket.org/lukaso25/udac-cs43198 .

The string description is "uDAC stereo audio DA converter UAC1 24/96" (slash can be omitted).

Selected VID and PID 0x1d50  0x616f.

Thank you.

Best regards

Lukáš Otava

cuvoodoo commented:
nice that you use the next sequential PID :). I'm trying to close a previously allocated gap though. Would you mind using PID 0x6155 instead ? Please edit/rebase the PR commit to use this PID (don't add a second commit).

the license file is present, but to be correctly applied, you also need to mention the license name in the schematic sheet (e.g. in the title block next to copyright author and date).
this is not required for board layouts though.

the schematic pdf export is present, but the board layout export gerber files are missing (see point 6 in our readme).

the KiCAD board layout opens fine, but not the schematic. It uses the symbols from the cache file instead of the proper library files (which seem to be present in the repo folder). Please fix the library setting so the project/schematic used the proper files.

lukaso25 commented:
Hello. Thank you for corrections. I have changed PID to 0x6155, no problem :-)

I have also updated schematic with licence info and correct library links. Gerber files are now included.

I hope everything is fine now.

Best regards, Lukáš Otava

cuvoodoo commented:
All is fine now. Good job.
